### DISCUSSION TITLE
Integrated Student Onboarding

### DISCUSSION THREAD

**Discussion by LaurenMcM749** [t3-oss/create-t3-turbo#1123] [Ideas]
### Describe the feature you'd like to request

 Application + Scheduling Interview + Team Matching + Scholarship + Payment - all one field in database. 
- Director of Ops is crucial in building this requirement
- Need to weigh pros/cons of building this or sticking with current Airtable solution, which is right now fulfilling our needs in a satisfactory way, with the exception of payment tracking. 
- After thoroughly presenting the problem, Ani can make proposal about an integrated solution and we can weigh pros/cons and prioritization

### Describe the solution you'd like to see

 Application + Scheduling Interview + Team Matching + Scholarship + Payment - all one field in database. 
*more details TBD

**Comment by dBianchii** [comment]
?
